source/includes/steps-install-shell-macos.yaml

```diff
@@ -23,5 +23,21 @@ source:
   file: steps-install-shell-base.yaml
   ref: add-shell-to-path
 ref: add-shell-to-path-macos
+---
+title: "Allow macOS to Run ``mongosh``."
+ref: remove-quarantine-macos
+level: 4
+content: |
+   macOS may prevent ``mongosh`` from running after installation. If
+   you receive a security error when starting ``mongosh`` indicating
+   that the developer could not be identified or verified, perform
+   the following actions:
+
+   a. Open *System Preferences*.
+   
+   #. Select the *Security and Privacy* pane.
 
+   #. Under the *General* tab, click the button to the right of the
+      message about ``mongod``, labelled either :guilabel:`Open Anyway`
```

Comment:
```suggestion
      message about ``mongosh``, labelled either :guilabel:`Open Anyway`
```

Reply:
Oops bad copy-paste. Thanks for flagging.